Signal/translations/en.lproj/Localizable.strings

```diff
@@ -518,7 +518,7 @@
 "FAILED_SENDING_BECAUSE_RATE_LIMIT" = "Too many failures with this contact. Please try again shortly.";
 
 /* action sheet header when re-sending message which failed because of untrusted identity keys */
-"FAILED_SENDING_BECAUSE_UNTRUSTED_IDENTITY_KEY" = "You must accept the new safety number before you will be able to send.";
+"FAILED_SENDING_BECAUSE_UNTRUSTED_IDENTITY_KEY" = "Recipient's safety number recently changed. You may wish to verify before resending.";
```

Comment:
"Recipient's safety number _has_ recently changed."?

Also I wonder if we should format this to replace `Recipient` with the "display name" of the recipient (especially in groups).